meteor/client/ui/ClockView/PresenterScreen.tsx

```diff
@@ -286,296 +281,308 @@ export const getPresenterScreenReactive = (props: RundownOverviewProps): Rundown
 	}
 }
 
-export class PresenterScreenBase extends MeteorReactComponent<
-	WithTiming<RundownOverviewProps & RundownOverviewTrackedProps & WithTranslation>,
-	RundownOverviewState
-> {
-	protected bodyClassList: string[] = ['dark', 'xdark']
+function PresenterScreenContent(props: WithTiming<RundownOverviewProps & RundownOverviewTrackedProps>): JSX.Element {
```

Comment:
perhaps rename `RundownOverviewProps` to `IProps` and `RundownOverviewTrackedProps` to `ITrackedProps`?

Reply:
I don't think `RundownOverviewTrackedProps` should be renamed as that is exported and used in other files

Reply:
renamed to `PresenterScreenProps` and `PresenterScreenTrackedProps`